README.md

```diff
@@ -1 +1,32 @@
-Endpoint	HTTP Method	CRUD Method	Result
+# TDD FASTAPI example
+
+## Environment Variables
+| Variable Name                    | Suggested development value        | Description
+|----------------------------------|------------------------------------|------------
+|STAGE                             | stage                              | Deployment stage
+|TESTING                           | 0                                  | Testing flag 
+|POSTGRES_USER                     | root                               | Postgres user name
+|POSTGRES_PASSWORD                 | password                           | Postgres password
```

Comment:
When making manual for .env file think that the person that will be trying to set up the app on thei machine know nothing about programming. So now they have no clue what they have to do with this. Add a word about making `.env` file and how it should look 1:1

Reply:
@czakuou Agree if this will be open to the world we should. But private repo could be without this step.

Reply:
@mkubasz agree but still wouldn't it be better to stick to good practices for the learning purposes?